Add drafts for RFC 6 & 7 (Ethereum defs and Ether HTLC) (#38)
comit-network/RFCs: RFC-003-SWAP-basic.md

| `beta_redeem_identity`  | `β::Identity`       | The identity on β that **B** will be transferred to when the β-HTLC is activated with the correct secret  |
| `secret_hash`           | `hex-encoded-bytes` | The output of calling `hash_function` on the secret                                                       |

> If `alpha_expiry` or `beta_expiry` are in the past to be invalid, implementations SHOULD consider the request invalid.

Author: FIx this!

> Implementations SHOULD consider messages with values of `alpha_expiry` or `beta_expiry` that are in the past to be invalid.

Reviewer: :wrench: Maybe we can move the important part of the sentence to the beginning?

```suggestion
Values of `alpha_expiry` or `beta_expiry` that refer to points in the past SHOULD be considered invalid by implementations.
```

| `beta_redeem_identity`  | `β::Identity`       | The identity on β that **B** will be transferred to when the β-HTLC is activated with the correct secret  |
| `secret_hash`           | `hex-encoded-bytes` | The output of calling `hash_function` on the secret                                                       |

> If `alpha_expiry` or `beta_expiry` are in the past, implementations SHOULD consider the request to be invalid.

Author: Because it's not important for the COMIT protocol that if someone sends you times in the past that you reject it with `SE`.
